manage

```diff
@@ -303,6 +303,9 @@ node.env() {
 
     (   set -e
 
+        build_msg DATA "searx_extra/update/update_pygments.py"
+        pyenv.cmd python searx_extra/update/update_pygments.py
+
```

Comment:
When do you suggest to update the pygments.less files ?
I guess when the pygments pypi package is updated ?
How not to forget about that ?
Because this is exactly the current situation: pygments.less files are out of sync with the pypi package.

Are git conflicts on these files a problem ? Nope. 
In case of conflicts: forget about the existing files, generated them, done.

This lead me to think that they are like the "node_modules" directories: a first step to build the theme sources, but instead of a HTTP download, it is a call to Python script.

Reply:
> I guess when the pygments pypi package is updated ?
> Because this is exactly the current situation: pygments.less files are out of sync with the pypi package.

OK, I understand; it is hard to automate the build of the less files when the python package is updated.

> This lead me to think that they are like the "node_modules" directories: 

IMO this is conceptual the wrong conclusion.  My questions are: 

- Do we update other /static files (which needs to be commited) when we build the target `node.env`? .. ASFAIK the answer would be 'no'.
- Do we nee node.env to create the LESS files? .. no, the less files are only related to the python packages, not related to node.

 > When do you suggest to update the pygments.less files ?

When the themes are build, when we build the targets `themes.oscar` and `themes.simple`.  Or with other words: the less files are not a part of the environment, the less files are a part of the themes.  To be precise: the less files are a prerequisite of the themes we commit into the repo somewhere under /src. 

- Since LESS files are only related to python, not to node, we need a target `pygments.less` to update these files.
-  `pygments.less` should be a prerequisite  of the themes.* targets

I already prepared a patch for this  .. should I push my patch on top of your branch?

Reply:
I think your d20f6a1f is the best solution .. there was a small typo I fixed in 1559ed38